CONTRIBUTING.md

```diff
@@ -36,28 +36,61 @@ information on using pull requests.
 
 #### Python
 
-The repository requires that files use the [Black](https://github.com/psf/black) code formatter and style.
+The repository requires that files use the [Black](https://github.com/psf/black)
+code formatter and style.
 
 #### Markdown
 
-The repository requires that files use the [Prettier](https://github.com/prettier/prettier) code formatter and style.
+The repository requires that files use the
+[Prettier](https://github.com/prettier/prettier) code formatter and style. For
+more information about Prettier's configuration for this project, see
+[the Prettier configuration file](/.prettierrc)
 
 #### Terraform
 
-We follow the guidelines and recommendations in the [Google Cloud Best practices for using Terraform](https://cloud.google.com/docs/terraform/best-practices-for-terraform) document, unless noted otherwise.
+We follow the guidelines and recommendations in the
+[Google Cloud Best practices for using Terraform](https://cloud.google.com/docs/terraform/best-practices-for-terraform)
+document, unless noted otherwise.
 
-The repository requires that files use built-in formatting using the `terraform fmt` command.
+The repository requires that files use built-in formatting using the
+`terraform fmt` command.
 
-#### Recommended Tools
+## Development environment
 
-##### Code OSS/VSCode Extensions
+To set up a development environment, we designed a
```

Comment:
This is all VSCode specific and won't work on Code OSS without "hacking" the marketplace.

Reply:
That should work if we change the wording some. Can you move this paragraph to the `Dev container configuration` section.

Reply:
Done.